package.json

```diff
@@ -17,20 +17,22 @@
     "uuid": "^8.3.2"
   },
   "devDependencies": {
-    "@types/jest": "27.4.1",
-    "@types/node": "17.0.27",
-    "@types/uuid": "8.3.4",
-    "@typescript-eslint/eslint-plugin": "5.21.0",
-    "@typescript-eslint/parser": "5.21.0",
-    "esbuild": "0.14.38",
-    "esbuild-jest": "0.5.0",
-    "eslint": "8.14.0",
-    "eslint-config-prettier": "8.5.0",
-    "eslint-plugin-prettier": "4.0.0",
-    "is-email": "1.0.2",
-    "jest": "28.0.1",
-    "prettier": "2.6.2",
-    "typescript": "4.6.3"
+    "@types/jest": "^27.4.1",
+    "@types/node": "^17.0.27",
+    "@types/uuid": "^8.3.4",
+    "@typescript-eslint/eslint-plugin": "^5.21.0",
+    "@typescript-eslint/parser": "^5.21.0",
+    "esbuild": "^0.14.38",
+    "esbuild-jest": "^0.5.0",
+    "eslint": "^8.14.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-prettier": "^4.0.0",
+    "is-email": "^1.0.2",
+    "is-mac-address": "^1.0.4",
+    "jest": "^28.0.1",
+    "prettier": "^2.6.2",
+    "typescript": "^4.6.3",
+    "user-agent-is-browser": "^0.1.0"
```

Comment:
Decided to change to caret ranges in this PR (we decided to use caret ranges, I just didn't change this yet). I know its technically unrelated to this PR, but adding it outside of this PR would cause conflicts, since this PR adds a bunch of dev deps. So seemed the lesser evil to just change it here.